tests/tests_password.yml

```diff
@@ -7,46 +7,51 @@
     mssql_accept_microsoft_cli_utilities_for_sql_server_eula: true
     mssql_accept_microsoft_sql_server_2019_standard_eula: true
   tasks:
-    - name: Include role to set up MSSQL Server
+    - name: Set up MS SQL
       include_role:
         name: linux-system-roles.mssql
       vars:
         mssql_password: "p@55w0rD"
         mssql_edition: Evaluation
 
-    - name: >
-        Include role to change the password.
-        Should report Changed.
-      include_role:
-        name: linux-system-roles.mssql
-      vars:
-        mssql_password: "p@55w0rD11"
+    - name: Modify the mssql-server service start limit interval
```

Comment:
add a comment here - something like "# otherwise the test restarts mssql-server too often for the default systemd settings"